components/playlist/playlist_service_helper.h

```diff
@@ -12,17 +12,60 @@
 
 namespace playlist {
 
-base::Value::Dict GetValueFromPlaylistItemInfo(const PlaylistItemInfo& info);
+// This class encapsulates how we convert data between mojo types and
+// base::Value. As base::Value's APIs are not that stable and quite confusing to
+// use, we'd like to strictly limit where it's done.
+class TypeConverter final {
```

Comment:
I don't think it makes sense to create a class that only has static methods, also if we did it should go in type_converter, not playlist_service_helper. I would convert these back to regular functions

Reply:
Agreed. Removed and rename files "type converter.h/cc"